plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/listeners/CodeWhispererCodeScanFileListener.kt

```diff
@@ -15,7 +14,8 @@ import software.aws.toolkits.jetbrains.services.codewhisperer.util.CodeWhisperer
 internal class CodeWhispererCodeScanFileListener(val project: Project) : EditorFactoryListener {
     override fun editorCreated(event: EditorFactoryEvent) {
         val actionManager = CodeWhispererExplorerActionManager.getInstance()
-        if (event.editor.virtualFile.isFile && event.editor.project == project && actionManager.isAutoEnabledForCodeScan() &&
```

Comment:
we could simply early return 
```
if (event.editor.virtualFile) {
  return
}
```

Reply:
given the amount of functionality that seems to indirectly need an editor, i think we should early return if the user didn't actually touch a file

Reply:
testing the early return now.